UI/api/api.tsx

```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+
+const client = new ApolloClient({
+    uri: "/api/graphql",
+    cache: new InMemoryCache(),
```

Comment:
in memory caches can cause issues if you're trying to load updated data from the api on each request - we might need to consider turning this off later.